cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/SvInjectionConversion.java

```diff
@@ -0,0 +1,91 @@
+import com.powsybl.iidm.network.*;
+import com.powsybl.triplestore.api.PropertyBag;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class SvInjectionConversion extends AbstractIdentifiedObjectConversion {
+
+    public SvInjectionConversion(PropertyBag p, Context context) {
+        super("SvInjection", p, context);
+        String tn = p.getId("TopologicalNode");
+        Terminal associatedTerminal = context.terminalMapping().findFromTopologicalNode(tn);
+        if (associatedTerminal == null) {
+            cgmesTerminal = context.cgmes().terminal(context.terminalMapping().findCgmesTerminalFromTopologicalNode(tn));
+            if (cgmesTerminal == null || context.cgmes().voltageLevel(cgmesTerminal) == null) {
+                context.invalid(id,
+                        String.format("The CGMES terminal and/or the voltage level associated to the topological node %s linked to the SV injection %s is missing",
+                                tn, id));
+                return;
+            }
+            voltageLevel = context.network().getVoltageLevel(context.cgmes().voltageLevel(cgmesTerminal));
+            if (context.nodeBreaker()) {
+                node = context.nodeMapping().iidmNodeForTerminal(cgmesTerminal, voltageLevel);
+            }
+        } else {
+            voltageLevel = associatedTerminal.getVoltageLevel();
+            if (context.nodeBreaker()) {
+                VoltageLevel vl = associatedTerminal.getVoltageLevel();
+                VoltageLevel.NodeBreakerView nb = vl.getNodeBreakerView();
+                node = nb.getNodeCount();
+                nb.setNodeCount(nb.getNodeCount() + 1);
+                vl.getNodeBreakerView().newInternalConnection()
+                        .setNode1(node)
+                        .setNode2(associatedTerminal.getNodeBreakerView().getNode())
+                        .add();
+            }
+        }
+        if (!context.nodeBreaker()) {
+            busId = context.namingStrategy().getId("Bus", tn);
+        }
+    }
+
+    @Override
+    public boolean valid() {
+        return voltageLevel != null
+                && ((context.nodeBreaker() && node != -1)
+                    || (!context.nodeBreaker() && voltageLevel.getBusBreakerView().getBus(busId) != null));
+    }
+
+    @Override
+    public void convert() {
+        LoadAdder adder = voltageLevel.newLoad()
+                .setP0(p.asDouble("pInjection"))
+                .setQ0(q0())
+                .setLoadType(LoadType.FICTITIOUS);
+        identify(adder);
+        if (context.nodeBreaker()) {
+            adder.setNode(node);
+        } else {
+            adder.setConnectableBus(busId)
+                    .setBus(busId);
+        }
+        Load load = adder.add();
+        if (cgmesTerminal != null) { // terminal only added if it has not been already added
+            context.terminalMapping().add(cgmesTerminal.id(), load.getTerminal(), 1);
```

Comment:
We could move the `convertedTerminal` function from `AbstractConductingEquipmentConversion` to the `Context` class and use it here.